applications/web3-open-jurisdiction-stack.md

```diff
@@ -0,0 +1,337 @@
+
+Having a form of Web3 jurisdiction would instil more [confidence](https://twitter.com/ph_lux/status/1646090060784193543?s=20) in investing that treasury, unlocking its growth potential.
+
+Similarly, the absence of a jurisdictional system today hinders enterprise participation. Enterprises require certainties, or at the very least, a neutral and reliable system to assess the achievement of milestones. Currently, it appears that the risk level associated with participating in grant releases is too high for well-structured organisations with fixed operational costs.
+
+These are just a few examples among many that highlight the necessity for every DAO or related construct to address the issue of a neutral third party capable of resolving disputes. This is an essential evolutionary step for fostering ecosystem growth.
+
+**Research questions/hypothesis.**
+
+We aim to conduct a comprehensive assessment of the jurisdictional requirements prevalent throughout the ecosystem (parachains, dApps, etc.). Specifically, we will analyse how Web3 treasuries can benefit from a dispute resolution system. Below, you will find a more detailed explanation of the questions and research outline.
+
+**The methodology that will be applied.**
+
+The majority of our research efforts will revolve around engaging with key stakeholders from Web3 Foundation, Parity, Polkadot, Kusama communities, and various parachain projects. These engagements will primarily take the form of recorded interviews, allowing us to meticulously document our findings.  
+The recorded interviews will produce one or more publicly available reports for anyone in the Web3 space.
+
+By taking this approach, we can ensure that the tech stack addresses the actual needs of ecosystem actors and dynamics, maximising the chances of concrete adoption.
+
+The reason why we think a thorough analysis phase (as per software engineering best practices) is needed in this case is because the impact of the software solution has a real world aspect to it aside from the involvement of several human stakeholders. We strongly advise against going heads down towards development as it is likely to be something of no use if not backed by data collected methodically and actual user interaction.
+
+To give an idea, although the actual questionnaire will be part of the _Milestone 1-1_, the discussion guide will be around questions such as:
+
+- Do you find the process of applying to grants fair and just?
+- Do you feel protected in case issues do arise while working on a grant?
+- Do you feel there should be a clear process for handling such issues?
+- Do you feel comfortable in allocating funds to entities?
+- Did it ever happen that you felt the outcome of a relationship with a treasury was unfair?
+- Do you think there is a need for a legally binding resolution method?
+- How would you currently resolve a disagreement with your counterparty?
```

Comment:
I want to clarify here that the W3F Grants Program is completely separate from treasury proposals. Can you specify whether you are referring to treasury proposals and bounties here?

Reply:
The discussion guide sample is regarding treasury proposals indeed 🙏  but just to be clear we would like to explore also grants flow of the Web3 Foundation (and others) to understand all the nuances.

Eventually the common abstraction between all these is the movement of funds from a common pot for a common good through a public initiative. We are interested in making this process as bulletproof as possible

Reply:
Thanks @0xlyd perhaps this is just a matter of confusing terminology. For example, I wouldn't call a successful treasury proposal a "grant". Maybe "spending request" or "treasury funds" would be more appropriate. Being that the W3F grants program is currently off-chain, comparing it with treasury proposals is kind of like comparing apples & oranges at this point. For example we do have contractual agreements with grantees, as well as terms & conditions, a contributor license agreement (CLA), private grants, etc. Essentially our program has some legal frameworks that the treasury might not have. I think this might create different considerations for each.

The W3F grants program also has a smaller scope. We mainly fund software and research, whereas the treasury has an expanded scope and can fund other things such as events and education initiatives. I see the treasury as more of a decentralized, open pot of "public funds".

It may seem pedantic, but in my mind the terminology is very important, as it could potentially distort the data if the participants are confused as to what you are referring to. The complexity of our ecosystem requires the need for explicit terms. If you could update your application to distinguish the difference between the areas of research I think this would help to avoid confusion.

Reply:
Thanks for your thorough answer @0xlyd sounds good! Feel free to ping me when I should take another look.